nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/EvaluateXPath.java

```diff
@@ -381,48 +380,6 @@ public void onTrigger(final ProcessContext context, final ProcessSession session
         }
     }
 
-    private void doTransform(final Source sourceNode, OutputStream out) throws TransformerFactoryConfigurationError, TransformerException {
-        final Transformer transformer;
-        try {
-            transformer = TransformerFactory.newInstance().newTransformer();
-        } catch (final Exception e) {
-            throw new ProcessException(e);
-        }
-
-        final Properties props = new Properties();
-        props.setProperty(OutputKeys.METHOD, "xml");
-        props.setProperty(OutputKeys.INDENT, "no");
-        props.setProperty(OutputKeys.OMIT_XML_DECLARATION, "no");
-        transformer.setOutputProperties(props);
-
-        final ComponentLog logger = getLogger();
-
-        final AtomicReference<TransformerException> error = new AtomicReference<>(null);
```

Comment:
Should StandardTransformProvider support the specification of an error listener?  I don't think we need to worry about a default implementation.

Reply:
Maybe also a test case exercising in StandardTransformProviderTest.

Reply:
That's a good question. This component was the only reference that had a custom `ErrorListener`. Given that the default `ErrorListener` throws exceptions on errors, that is the desired standard behavior. Supporting a custom `ErrorListener` might be useful in future use cases, but it also opens the door to hiding transform processing problems. For that reason, it seems best to avoid supporting a custom `ErrorListener` for now, and address it later if a different use case arises.